clearwater-infrastructure/usr/share/clearwater/bin/poll-http

```diff
@@ -2,7 +2,7 @@
 
 # @file poll-http
 #
-# Copyright (C) Metaswitch Networks 2016
+# Copyright (C) Metaswitch Networks 2017
```

Comment:
Don't change this - this year tracks when the file was created, not when it was last updated (same comment throughout).

Reply:
Fixed.